k8s/apis/fabric/v1/webhook_suite_test.go

```diff
@@ -83,12 +83,8 @@ var _ = BeforeSuite(func() {
 	// start webhook server using Manager
 	webhookInstallOptions := &testEnv.WebhookInstallOptions
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
-		Scheme:             scheme,
```

Comment:
Is this because controller-runtime package upgrade?

Reply:
Yes